packages/wrangler/src/__tests__/provision.test.ts

```diff
@@ -30,6 +31,7 @@ describe("--x-provision", () => {
 	const { setIsTTY } = useMockIsTTY();
 
 	beforeEach(() => {
+		logger.loggerLevel = "debug";
```

Comment:
You might need to restore the loggerLevel after the test or it will create a side effect for the rest of the tests.